lib/x11test.pm

```diff
@@ -1137,4 +1138,98 @@ sub cleanup_firefox_print {
     send_key 'ctrl-d';
     assert_screen 'generic-desktop';
 }
+
+# MM network check: try to ping the gateway, the client and the internet
+sub ensure_client_reachable {
+    assert_script_run('ping -c 1 10.0.2.2');
+    assert_script_run('ping -c 1 10.0.2.102');
+    assert_script_run('curl conncheck.opensuse.org');
+}
+
+sub x11_server_preparation {
```

Comment:
Those preparations are dedicated for a podman workload and should as such not be in the generic `x11test` library function. Please move them either to their own module in `lib/` or back to the test module.

The same also applied to the next subroutines - IMHO are they not generic enough for `x11test` and should have their own library module.